/* バージョン */
syntax = "proto3";

/* パッケージ */

/* インポート */

/* サービス */
service UidService {
  /* RPC */
  rpc GetUidPool (RequestUidPoolSize) returns (ResponseUidPool) {}
}

/* メッセージ型 */
message RequestUidPoolSize {
  /* メンバ変数 */
  fixed64 size = 1;
}

message ResponseUidPool {
  fixed64 start_uid = 1;
  fixed64 end_uid = 2;
}
